rhel/vex/fetcher.go

```diff
@@ -196,6 +196,11 @@ func (u *Updater) Fetch(ctx context.Context, hint driver.Fingerprint) (io.ReadCl
 // to w means they are deemed to have changed since the compressed
 // file was last processed. w and fp can be modified.
 func (u *Updater) processChanges(ctx context.Context, w io.Writer, fp *fingerprint) error {
+	tf, err := tmp.NewFile("", "rhel-vex-changes.")
+	if err != nil {
+		return err
+	}
```

Comment:
Need to defer the close, otherwise the file leaks at any early return.

Reply:
oh dur